ethcore/src/snapshot/account.rs

```diff
@@ -62,53 +63,50 @@ impl CodeState {
 
 // walk the account's storage trie, returning an RLP item containing the
 // account properties and the storage.
-pub fn to_fat_rlp(acc: &BasicAccount, acct_db: &AccountDB, used_code: &mut HashSet<H256>) -> Result<Bytes, Error> {
+pub fn to_fat_rlps(acc: &BasicAccount, acct_db: &AccountDB, used_code: &mut HashSet<H256>, max_storage_items: usize) -> Result<Vec<Bytes>, Error> {
```

Comment:
what will happen if the current chunk is half-full and the first `Bytes` in the returned `Vec` doesn't fit into the 16MB limit? does it just get closed like that and the next chunk it used?